src/Integration/ErrorListenerIntegration.php

```diff
@@ -58,7 +58,7 @@ public function setupOnce(): void
                 return;
             }
 
-            $currentHub = Hub::getCurrent();
+            $currentHub = SentrySdk::getCurrentHub();
```

Comment:
this is still binding *any* integration to the "current" hub.

What should be done would be to inject the hub as a constructor argument (if null, then fetch the current hub from the sentry sdk as bc).

Reply:
We could inject the `HubAdapter` in the constructor (it has been coded for cases as this one), however if I don't recall wrong binding an integration to a specific hub does not play well with the current architecture, or at least it doesn't work well with all the integrations provided out of the box. I'm sure @HazAT can give a better explanation of why it isn't a so easy task to make this change, so maybe it's better to discuss about this in a separate issue. The purpose of this PR is mainly to move away from the `Hub` class the storage of the instance of the current hub